### ISSUE TITLE
Insert on tables with triggers fails in 1.1.3

### ISSUE THREAD

**Issue by timnyborg** [microsoft/mssql-django#130]
As of 1.1.3, when I attempt to insert rows on tables with triggers through a standard model.save(), I receive the error "The target table 'email' of the DML statement cannot have any enabled triggers if the statement contains an OUTPUT clause without INTO clause."

I assume this is due to https://github.com/microsoft/mssql-django/pull/107, which has replaced the SCOPE_IDENTITY with an OUTPUT INSERTED.id

**Comment by jmah8** [comment]
Thanks for the report. I can reproduce this issue.

**Comment by jmah8** [comment]
Closing this issue since a workaround for this error was introduced in PR #158. If there is a trigger in a database you can add "has_trigger": True to the OPTIONS dictionary eg:
```    
DATABASES = {
        'default': {
            'ENGINE': 'mssql',
	     ...
            'OPTIONS': {
                'driver': 'ODBC Driver 17 for SQL Server',
		'has_trigger': True,
            },
        },
    }
```
It is false by default so if other databases don't have triggers nothing needs to be added. This workaround prevents returning rows from bulk insert when `has_trigger = True` though.

**Comment by jmah8** [comment]
Some updates regarding this issue, after some internal discussion we decided to change the workaround so that inserting into tables with triggers follows the behaviour from v1.1.2 (works by default). So instead of `has_trigger` we replaced it with `return_rows_bulk_insert` as an option in the `OPTIONS` dictionary. It is false by default and when false it allows for inserting into tables with triggers with no errors but prevents returning ids from bulk insert. The PR that has these changes is PR #161 and will be included in the next release.
